src/components/buttons/accessible-button/accessible-button.js

```diff
@@ -28,6 +28,7 @@ const AccessibleButton = React.forwardRef((props, ref) => (
       props.isDisabled &&
         css`
           cursor: not-allowed;
+          pointer-events: none;
```

Comment:
otherwise it swallows up events

Reply:
We should leave a comment to explain the reasons

Reply:
Maybe instead more examples of uses with different elements would be useful>?

Reply:
Sure, but also I mean for future references when someone looks at this code and it might not be clear what the reason for disabling pointer events is.